tests/Unit/Evolution/Systems/Cce/Test_NewmanPenrose.cpp

```diff
@@ -24,8 +37,316 @@ void pypp_test_volume_weyl() {
+
+  db::mutate<spin_weighted_variables_tag>(
+      make_not_null(&box),
+      [&spatial_metric_coefficients, &dt_spatial_metric_coefficients,
+       &dr_spatial_metric_coefficients, &shift_coefficients,
+       &dt_shift_coefficients, &dr_shift_coefficients, &lapse_coefficients,
+       &dt_lapse_coefficients, &dr_lapse_coefficients, &extraction_radius](
+          const gsl::not_null<Variables<spin_weighted_boundary_tags>*>
+              spin_weighted_boundary_variables) {
+        create_bondi_boundary_data(
+            spin_weighted_boundary_variables, spatial_metric_coefficients,
+            dt_spatial_metric_coefficients, dr_spatial_metric_coefficients,
+            shift_coefficients, dt_shift_coefficients, dr_shift_coefficients,
+            lapse_coefficients, dt_lapse_coefficients, dr_lapse_coefficients,
+            extraction_radius, l_max);
+      });
+
+  // construct the coordinate transform quantities
+  const double variation_amplitude_inertial = value_dist(*gen);
+  db::mutate<Tags::CauchyCartesianCoords>(
+      make_not_null(&box),
+      [&l_max](const gsl::not_null<tnsr::i<DataVector, 3>*>
+                                 cauchy_cartesian_coordinates) {
+        tnsr::i<DataVector, 2> cauchy_angular_coordinates{
+            get<0>(*cauchy_cartesian_coordinates).size()};
+        // There is a bug in Clang 10.0.0 that gives a nonsensical
+        // error message for the following call to
+        // cached_collocation_metadata unless l_max is captured in
+        // this lambda.  (The capture should not be necessary.)  This
+        // line silences the (correct) "unused capture" warnings.
+        (void)l_max;
+        const auto& collocation = Spectral::Swsh::cached_collocation_metadata<
+            Spectral::Swsh::ComplexRepresentation::Interleaved>(l_max);
+        for (const auto collocation_point : collocation) {
+          get<1>(cauchy_angular_coordinates)[collocation_point.offset] =
+              collocation_point.phi;
+          get<0>(cauchy_angular_coordinates)[collocation_point.offset] =
+              collocation_point.theta;
+        }
+        get<0>(*cauchy_cartesian_coordinates) =
+            sin(get<0>(cauchy_angular_coordinates)) *
+            cos(get<1>(cauchy_angular_coordinates));
+        get<1>(*cauchy_cartesian_coordinates) =
+            sin(get<0>(cauchy_angular_coordinates)) *
+            sin(get<1>(cauchy_angular_coordinates));
+        get<2>(*cauchy_cartesian_coordinates) =
+            cos(get<0>(cauchy_angular_coordinates));
+      });
+
+  db::mutate<Tags::PartiallyFlatCartesianCoords>(
+      make_not_null(&box),
+      [&l_max, &variation_amplitude_inertial](
+          const gsl::not_null<tnsr::i<DataVector, 3>*>
+              inertial_cartesian_coordinates) {
+        tnsr::i<DataVector, 2> inertial_angular_coordinates{
+            get<0>(*inertial_cartesian_coordinates).size()};
+        // There is a bug in Clang 10.0.0 that gives a nonsensical
```

Comment:
sorry, why would `l_max` not need to be captured? It's used on like 249 and is not a `constexpr` variable.